server/manager/src/main/java/org/apache/accumulo/manager/split/Splitter.java

```diff
@@ -112,10 +102,7 @@ private static int weigh(KeyExtent keyExtent) {
     return size;
   }
 
-  public Splitter(ServerContext context, Ample.DataLevel level, Manager manager) {
-    this.context = context;
-    this.level = level;
-    this.manager = manager;
+  public Splitter(ServerContext context) {
     this.splitExecutor = context.threadPools().createExecutorService(context.getConfiguration(),
         Property.MANAGER_SPLIT_WORKER_THREADS, true);
     this.scanExecutor =
```

Comment:
This thread pool may be able to be removed

Reply:
Addressed in cfced0b